source/geod24/concurrency.d

```diff
@@ -2126,3 +2452,177 @@ public:
         }
     );
 }
+
+@system unittest
```

Comment:
`@system` is already the default though.

Reply:
Most of the tests in `concurrency.d` already have it. Lets keep it?

Reply:
Yeah